### DISCUSSION TITLE
Unable to Get Vanjaro to Work with DNN 9.9.1

### DISCUSSION THREAD

**Discussion by mayaco** [vanjarosoftware/Vanjaro.Platform#1614] [Q&A]
Hello,
After creating a clean DNN 9.9.1 installation and installing the DNN Vanjaro 1.1.0 extension I'm unable to create a new page with Vanjaro. The Vanjaro Editor does not display and only a blank screen appears.  Here's a short video showing what I'm experiencing. https://www.screencast.com/t/hYfIz2LijF

Any ideas why the Vanjaro editor would not display?

**Comment by mksmanish79** [comment]
We are trying to duplicate the issue. Is it working after refreshing the page?

**Reply by mayaco** [reply]
No it is not working after refreshing the page.

**Comment by mksmanish79** [comment]
We are able to duplicate the issue and will be fixed in next update. 

For now the workaround is

- Refresh the page that is opened after create new page click and it will work. 
- Try again to create the page and it will work on second attempt.

**Reply by aloscher** [reply]
I re-installed and all works perfect now. Just my pixabay api key will not take. says Input is not a valid Pixabay API Key when it is valid

**Reply by mayaco** [reply]
I tried your suggestion and after refreshing the page I'm still not able to create a new page. The page is still blank. I'm using Firefox and I tried the steps in a private tab and it did not work for me.

**Reply by mayaco** [reply]
I also re-installed the Vanjaro extension (used repair option) and it still did not work for me. I started fresh with a new DNN installation and repeated the steps with the Vanjaro extension and I'm getting the same results.

**Reply by mayaco** [reply]
Will there be a new release coming out soon? I'm still not able to get Vanjaro_For_DNN_1.1.0_x64_Install to work on DNN 9.9.1+.
